### PR TITLE
fix anchor highlight in darkmode

### PR DESCRIPTION
**Pull request by Lupus590**
a nice dark blue

### CODE REVIEW

File: src/doc_emit/html_embedded_styles.css [hunk 1]
```diff
@@ -114,6 +114,9 @@ ul ol { margin-top: 0px; }
 
 /* Make the target distinct; helps when we're navigating to a function */
 a:target + * { background-color: #FFFF99; }
+@media (prefers-color-scheme: dark) {
```
Comment (SquidDev) [inline src/doc_emit/html_embedded_styles.css:117]: This should really be using CSS variables (and thus defined at the top of the file) like the rest of the colour scheme.

**Review by SquidDev — CHANGES_REQUESTED** [review]

File: src/doc_emit/html_embedded_styles.css (hunk 1 above)
Comment (Lupus590) [reply src/doc_emit/html_embedded_styles.css:117]: Shows how much I know about CSS.

**Review by SquidDev — CHANGES_REQUESTED** [review]
Oh, no. Sorry. So one needs something like this:

```css
:root {
  --highlight-header-colour: #FFFF99;
}
/* Same as above for dark mode. */

 a:target + * { background-color: var(--highlight-header-colour); }
```

**Comment by Lupus590** [comment]
That makes a lot more sence.

**Review by SquidDev — APPROVED** [review]
> a nice dark blue

This isn't dark blue. It's just black.

**Comment by Lupus590** [comment]
it looked dark blue when I was testing it, maybe my monitor needs to be calibrated. Or my eyes.

**Pull request merged**